lib/raven/configuration.rb

```diff
@@ -311,7 +311,10 @@ def capture_in_current_environment?
     end
 
     def server_configured?
-      host && path
+      verify!
```

Comment:
Wouldn't it be better to make this method without using `rescue`?  According to the fast-ruby lecture I got to know due to @nateberkopec this pattern not only looks IMO bad also seems to be quite slow. `capture_allowed?` uses this method and it seems to be called quite often. 

I would propose the following:

1. Move the needed settings to a final variable (i.e. the `%w(...)` )
2. Create a method that returns the index of the first missing setting. (e.g.: something like `firstMissingSetting`)
3. Adapt `verify!` and `server_configured?` to use this new `firstMissingSetting`.

PS - Thx @clupprich for this feature!